internal/cosesign1/create.go

```diff
@@ -0,0 +1,134 @@
+package cosesign1
+
+import (
+	"crypto/rand"
+
+	"crypto"
+	"crypto/x509"
+	"encoding/pem"
+	"io"
+	"log"
+
+	"github.com/veraison/go-cose"
+)
+
+func pem2der(chainPem []byte) []byte {
+	block, rest := pem.Decode(chainPem)
+	var r = []byte{}
+	if block.Bytes != nil {
+		r = block.Bytes
+	}
+	for len(rest) != 0 {
+		block, rest = pem.Decode(rest)
+		if block.Bytes != nil {
+			r = append(r, block.Bytes...)
```

Comment:
consider using `bytes.Buffer`.

Reply:
I have saved an init line if there is no block and will return nil rather than an empty array but I don't see how a buffer is better than just append.